cdp_scrapers/scraper_utils.py

```diff
@@ -188,42 +188,33 @@ def parse_static_file(file_path: Path) -> ScraperStaticData:
     Function looks for "seats", "primary_bodies", "persons" top-level keys
     """
     with open(file_path) as static_file:
-        static_json: Dict[str, Dict[str, Any]] = json.load(static_file)
-
-        if "seats" not in static_json:
-            seats: Dict[str, Seat] = {}
-        else:
-            seats: Dict[str, Seat] = {
-                seat_name: Seat.from_dict(seat)
-                for seat_name, seat in static_json["seats"].items()
-            }
-
-        if "primary_bodies" not in static_json:
-            primary_bodies: Dict[str, Body] = {}
-        else:
-            primary_bodies: Dict[str, Body] = {
-                body_name: Body.from_dict(body)
-                for body_name, body in static_json["primary_bodies"].items()
-            }
-
-        if "persons" not in static_json:
-            known_persons: Dict[str, Person] = {}
-        else:
-            known_persons: Dict[str, Person] = {
-                person_name: parse_static_person(person, seats, primary_bodies)
-                for person_name, person in static_json["persons"].items()
-            }
-
+        static_json = json.load(static_file)
```

Comment:
I was sort of hoping that this entire thing could become a single dataclass json call:

```python
with open(file_path, "r") as static_file:
    return ScraperStaticData.from_json(static_file.read())
```

but I think I see from your other comments that the format of the file isn't quite setup for this behavior.

I will defer to @dphoria here for opinions since that file is largely his spec.

Reply:
It would have been really nice if the input data was always standardized and matched Class specs. We could have made some fields optional downstream, but that may affect contracts with other pieces of code.

May be there is a way to sanitize and/or provide default values before any kind of deserialization is taken up.

Reply:
> I was sort of hoping that this entire thing could become a single dataclass json call:
> 
> ```python
> with open(file_path, "r") as static_file:
>     return ScraperStaticData.from_json(static_file.read())
> ```
> 
No disagreement there. I think my original reason in making these files not 100% serialized dumps of the associated `IngestionModel` classes was because that would have made the files much bigger, though at this point I am going off memory so can't say for certain. I like that idea as well, so @rajataggarwal91 , if you want to make a separate MR to just modify these 'static data' files full serializations, and get that merged in first (or at least investigate feasibility of doing so), I think that would be better going forward as well. Up to you.

Reply:
Oh, another alternative is you/I can investigate/implement the changes after this PR also.

I prefer not to include that in this PR. I prefer keeping PRs small, if possible...

Reply:
No problems. I can modify all required static files to ensure they can be seamlessly deserialized without additional steps. I will make that change part of this PR.